### PR TITLE
Set version 2.5.2-a-1. 

### PR DESCRIPTION
**Pull request by Shelob9**
Also update changleog and stable tag to reflect existence of 2.5.1.1

#2635

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,4 +46,4 @@ We're able to focus more on Pods when we have some cash to help keep the lights
 
 <em>Please submit all pull requests against [release/3.0](https://github.com/pods-framework/pods/tree/release/3.0) branch.</em>
 
-~Current Version: 2.5.1~
+~Current Version: 2.5.1.1~
```
Comment (sc0ttkclark) [inline README.md]: 2.5.2-a-1?

File: readme.txt [hunk 2]
```diff
@@ -4,7 +4,7 @@ Donate link: http://podsfoundation.org/donate/
 Tags: pods, custom post types, custom taxonomies, user fields, custom fields, cck, cms, content types, database, framework, drupal, post types, avatars, comment fields, media fields
 Requires at least: 3.8
 Tested up to: 4.1.1
-Stable tag: 2.5.1
+Stable tag: 2.5.1.1
```
Comment (sc0ttkclark) [inline readme.txt]: 2.5.2-a-1

Comment (Shelob9) [inline readme.txt]: 2.5.1.1 is the current stable release on WPORG

Comment (sc0ttkclark) [inline readme.txt]: gotcha

File: README.md (hunk 1 above)
Comment (Shelob9) [inline README.md]: What exactly is the purpose of this line supposed to be? Current release version or current dev version? I suspect the answer is we should have both as their own line.

Comment (sc0ttkclark) [inline README.md]: On 2.x, it acts as the current dev version (should always mirror init.php of the branch) for the GitHub plugin updater. When we update that script via #2636 we won't have to even have this here anymore as it'll pull from the plugin headers inside init.php

Comment (sc0ttkclark) [inline README.md]: Never made this change / merged it, probably should.

Comment (Shelob9) [inline README.md]: Shoot, I thought you merged #2641 already
#gitflowfail

**Pull request merged**